templates/features.html

```diff
@@ -88,7 +100,7 @@
 
   var xhr = new XMLHttpRequest();
   xhr.open('GET', url);
-  //xhr.responseType = 'json'; // Safari 6.0.5 doesn't support this
+  xhr.responseType = 'json';
   xhr.onloadend = function(e) {
     var resp = e.target.response;
     featureList.features = (typeof resp == 'string') ? JSON.parse(resp) : resp;
```

Comment:
You can remove this now if `responseType` is set to `json` and simply have `featureList.features = e.target.response`